sdk/hdinsight/Azure.ResourceManager.HDInsight/CHANGELOG.md

```diff
@@ -1,15 +1,19 @@
 # Release History
 
-## 1.2.0-beta.5 (Unreleased)
+## 1.2.0-beta.5 (2025-05-07)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+- Upgraded api-version tag from 'package-2024-08-preview' to 'package-2025-01-preview'. Tag detail available at https://github.com/Azure/azure-rest-api-specs/blob/4c0f7731c93696af01bd2bb9927bf28d2afcbc98/specification/hdinsight/resource-manager/readme.md.
+    - Support for setting Entra User during HDInsight cluster creation.
+    - Support to add or update Entra UserInfo for an Entra-enabled HDInsight cluster
```

Comment:
Support to update Entra User Information in an existing Entra User enabled HDInsight cluster